### DISCUSSION TITLE
hoppscotch vs Others

### DISCUSSION THREAD

**Discussion by udaya-reddy** [github/india#39] [Open Source] [answered]
How is hoppscotch different from other tools like Postman?

**Comment by dhruvg20** [comment]
Hi @liyasthomas, request if you could help here 🙇🏽

**Comment by liyasthomas** [comment, marked as answer]
Hoppscotch embrace open source at it's core. Every project including the web client, proxy servers, browser extensions, CLI tool, API documentation generator etc are fully open sourced. Apart from testing RESTful APIs, Hoppscotch support GraphQL APIs, WebSocket, SSE, Socket.IO, MQTT brokers etc. Hoppscotch is a self-hostable 100% client side application - which means nothing goes out of your browser localstorage. The whole web client can be hosted from a Docker container for in any static hosting services like GitHub pages, Netlify, Vercel etc.

Follow the project to know all of it's features: https://github.com/hoppscotch/hoppscotch

**Comment by FRTMentor** [comment]
Hoppscotch embrace open source at it's core. Every project including the web client, proxy servers, browser extensions, CLI tool, API documentation generator etc are fully open sourced. Apart from testing RESTful APIs, Hoppscotch support GraphQL APIs, WebSocket, SSE, Socket.IO, MQTT brokers etc. Hoppscotch is a self-hostable 100% client side application - which means nothing goes out of your browser localstorage. The whole web client can be hosted from a Docker container for in any static hosting services like GitHub pages, Netlify, Vercel etc.

Follow the project to know all of it's features: https://github.com/hoppscotch/hoppscotch
